src/audio/host.c

```diff
@@ -789,7 +789,7 @@ static int host_copy(struct comp_dev *dev)
 	 */
 	if (pipeline_is_preload(dev->pipeline)) {
 		ret = dma_copy(hd->dma, hd->chan, hd->dma_buffer->size,
-			       DMA_COPY_PRELOAD);
+			       DMA_COPY_PRELOAD | DMA_COPY_BLOCKING);
```

Comment:
Can you explain in commit message why we need blocking copy mode and how it works.

Reply:
Done.